scene/gui/popup_menu.cpp

```diff
@@ -1099,6 +1099,7 @@ void PopupMenu::remove_item(int p_idx) {
 
 	items.remove(p_idx);
 	update();
+	minimum_size_changed();
```

Comment:
Is this also needed for the `add_separator` below?

Reply:
I think this only needs to happen when the size of the pop-up becomes smaller, when it grows it does something else entirely. I'll check again though

Reply:
Oh, you're right of course, when it grows you don't need to update the *minimum* size indeed.